XenModel/Actions/VM/VMCrossPoolMigrateAction.cs

```diff
@@ -41,9 +40,10 @@ public class VMCrossPoolMigrateAction : PureAsyncAction
     {
         private readonly VmMapping mapping;
         private readonly XenAPI.Network transferNetwork;
+        private readonly bool copy;
 
-        public VMCrossPoolMigrateAction(VM vm, Host destinationHost, XenAPI.Network transferNetwork, VmMapping mapping)
-            : base(vm.Connection, GetTitle(vm, destinationHost))
+        public VMCrossPoolMigrateAction(VM vm, Host destinationHost, XenAPI.Network transferNetwork, VmMapping mapping, bool copy)

```

Comment:
Bool copy does mean Copy or Migrate mode here. I think an Enum or a different action would be more clear here. (Especially copy = false does not suggest migrate to someone not familiar in this context)